include/ck/tensor_operation/gpu/device/device_batched_gemm_multi_d.hpp

```diff
@@ -53,6 +53,47 @@ struct DeviceBatchedGemmMultiD : public BaseOperator
     virtual std::unique_ptr<BaseInvoker> MakeInvokerPointer() = 0;
 };
 
+template <typename ALayout,
+          typename BLayout,
+          typename DsLayout,
+          typename ELayout,
+          typename ADataType,
+          typename BDataType,
+          typename DsDataType,
+          typename EDataType,
+          typename AElementwiseOperation,
+          typename BElementwiseOperation,
+          typename CDEElementwiseOperation>
+struct DeviceBatchedGemmV2MultiD : public BaseOperator
```

Comment:
What is the diffrence between this Device op and DeviceBatchedGemmMultiD?

Reply:
It is based on a different gridwise_gemm, which has different template parameters.

Reply:
Sure but why we nee diffrenet "DeviceBatchedGemm" for them? Can DeviceBatchedGemmMultiD_Xdl_CShuffle_V3 be base class for DeviceBatchedGemmMultiD?